news/2018-04-12-new-featured-artist-dictate.md

```diff
@@ -0,0 +1,105 @@
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/419.mp3" type="audio/mpeg">
+</audio>
+
+> A gritty prowl throughout an industrial soundscape, Militant pads throughout a slow build that gradually expands in complexity at a relatively even intensity, with plenty of room to support wild jumps and experimentation with distance.
+
+### Scavenger
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/420.mp3" type="audio/mpeg">
+</audio>
+
+> Tentative and foreboding, Scavenger starts off slow before exploding into a harsh, dominating and bassy synth, all underset with the twisted growling of a dark rumble.
+
+### Cauldron 
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/415.mp3" type="audio/mpeg">
+</audio>
+
+> Accompanied by shifting warble and stuttering percussion, Cauldron all but breathes in a bassy rumble set to synth hits that are constantly changing and shifting.
+
+### Shaman
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/421.mp3" type="audio/mpeg">
+</audio>
+
+> As ominous as the name might suggest, Shaman is underpinned by sharp, striking percussion that hits you hard and fast. Varied but ever building, the track provides an excellent platform for experimenting with changes in speed and patterning.
+
+### Empire
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/416.mp3" type="audio/mpeg">
+</audio>
+
+> Starting small and rapidly building to a frantic peak, Empire revels in the distant song of faded synths and padwork, spurned to a grungy sprint by unrelenting bass and percussion.
+
+### Melodik
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/417.mp3" type="audio/mpeg">
+</audio>
+
+> Contemplative but ever building, Melodik repeats but does not bore, breakbeat-tinged percussion fighting with grand bassy hits for prominence.
+
+*A short version of this track is also available.*
+
+### Slug
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/422.mp3" type="audio/mpeg">
+</audio>
+
+> Slower paced but no less ominous, Slug features sweeping synths framed by encroaching bass.
+
+### Treason
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/423.mp3" type="audio/mpeg">
+</audio>
+
+> Heavily breakbeat and drum 'n bass themed, Treason lurches backwards and forwards, but never stops building, endlessly ominous, and endlessly gritty.
+
+### Temple
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/424.mp3" type="audio/mpeg">
+</audio>
+
+> Dark, looming synths are peeled bare by percussive noise into a tribal themed beat, all beset by Dictate's classic mastery of bass.
+
+-------
+
+All the tracks listed above (and on [his Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/21)) are available **now** for mapping, totally free for use in osu!. Worry less, and map more!
```

Comment:
listing -> page

Reply:
we've used listing for literally every other FA announcement since the project started